html/README.html

```diff
@@ -180,6 +176,5 @@ <h2 id="donations">Donations</h2>
 Bitcoin is cool too: 
     1GrnYwUCAsQY3oejdu8CRutnG55Wi7bXHz  
```

Comment:
We should totally change this to a coinbase address. Or something more centralized.

Reply:
:octocat: